src/Downloader.ts

```diff
@@ -586,19 +591,34 @@ class Downloader {
   }
 
   private async getCompressedBody(resp: any): Promise<any> {
-    return this.isMimeTypeImage(resp.headers['content-type']) ? await imagemin.buffer(resp.data, imageminOptions) : resp.data;
+    if (isBitmapImageMimeType(resp.headers['content-type'])) {
+      if (isWebpCandidateImage(this.webp, this.cssDependenceUrls.hasOwnProperty(resp.config.url), resp.headers['content-type'])) {
```

Comment:
`this.cssDependenceUrls.hasOwnProperty(resp.config.url)` should not be given as argument, but just part of the test:
```
isWebpCandidateImage(this.webp, resp.headers['content-type']) &&
  !this.cssDependenceUrls.hasOwnProperty(resp.config.url)
```